src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig

```diff
@@ -187,6 +177,32 @@
             </div>
         </div>
 
+        <div class="tab {{ collector.requestcookies.all is empty and collector.responsecookies.all is empty ? 'disabled' }}">
+            <h3 class="tab-title">Cookies</h3>
+
+            <div class="tab-content">
+                <h3>Cookies Received</h3>
```

Comment:
Just asking: would the following changes make this easier to understand?

Instead of `Cookies Received` -> `Request Cookies` and
instead of `Cookies Created` -> `Response Cookies`